src/Build.OM.UnitTests/Instance/ProjectTargetInstance_Tests.cs

```diff
@@ -130,6 +131,29 @@ public void FileLocationAvailableEvenAfterEdits()
             }
         }
 
+        [Fact]
+        public void ProjectTargetsAreEnumerableByName()
```

Comment:
Another viable location for this test might be [Project_Tests](https://github.com/Microsoft/msbuild/blob/master/src/Build.OM.UnitTests/Definition/Project_Tests.cs). Though that class is already bloated.